### PR TITLE
Address rules fps may

### PR DESCRIPTION
**Pull request by mstemm**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

/kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

/area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
rule(macro trusted_logging_images): Add addl fluentd image
rule(macro trusted_logging_images): Let azure-npm image write to /var/log
rule(macro lvprogs_writing_conf): Add lvs as a lvm program
rule(macro user_known_k8s_client_container): Allow hcp-tunnelfront to run kubectl in containers
rule(list allowed_k8s_users): Add vertical pod autoscaler as known k8s users
```

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2687,8 +2689,13 @@
 # Whitelist for known docker client binaries run inside container
 # - k8s.gcr.io/fluentd-gcp-scaler in GCP/GKE
 - macro: user_known_k8s_client_container
-  condition: (k8s.ns.name="kube-system" and container.image.repository=k8s.gcr.io/fluentd-gcp-scaler)
- 
+  condition: >
+    (k8s.ns.name="kube-system" and container.image.repository=k8s.gcr.io/fluentd-gcp-scaler) or
+    container.image=mcr.microsoft.com/aks/hcp/hcp-tunnel-front
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2694]: container.image -> `container.image.repository`

File: rules/falco_rules.yaml [hunk 2]
```diff
@@ -2687,8 +2689,13 @@
...
+
+- macro: user_known_k8s_client_container_parens
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2696]: is this macro used by any rule?

File: rules/falco_rules.yaml (hunk 1 above)
Comment (mstemm) [reply rules/falco_rules.yaml]: Thanks, fixed.

File: rules/falco_rules.yaml (hunk 2 above)
Comment (mstemm) [reply rules/falco_rules.yaml:2696]: Oh sorry, I was editing things and missed this. I wanted to replace the use of user_known_k8s_client_container in the rule's condition. Fixed.

**Pull request merged**